receiver/receivercreator/observerhandler.go

```diff
@@ -182,10 +190,10 @@ func (obs *observerHandler) OnRemove(removed []observer.Endpoint) {
 		}
 
 		for _, rcvr := range obs.receiversByEndpointID.Get(e.ID) {
-			obs.params.TelemetrySettings.Logger.Info("stopping receiver", zap.Reflect("receiver", rcvr), zap.String("endpoint_id", string(e.ID)))
+			obs.logger.Info("stopping receiver", zap.Reflect("receiver", rcvr), zap.String("endpoint_id", string(e.ID)))
```

Comment:
Probably can peel off this logger change in a separate PR just to make the diff smaller